doc/news/changes/minor/20230518Heinz

```diff
@@ -0,0 +1,3 @@
+New: The new function distributed_compute_intersection_locations() computes intersections on parallel::distributed::Triangulations using CGAL. It returns a new class DistributedComputeIntersectionLocationsInternal which stores the found intersections and relevant information about communication. The class contains a helper function convert_to_distributed_compute_point_locations_internal() to produce DistributedComputePointLocationsInternal without an additional (possibly expensive) search of points and can therefore be used to initialize RemotePointEvaluation.
```

Comment:
Could you wrap after 80 characters?

Reply:
[c279452](https://github.com/dealii/dealii/pull/15165/commits/c27945208f374ebc93df5312f47ad3219f9383d4)